### PR TITLE
refactor apiCall to add specification to the fetchRandom, updated css…

### PR DESCRIPTION
**Pull request by AshleeAWebb**
… for homepage and random so images fit within a container

# PR Template

### Types of changes made?

- [ ]  testing
- [ ]  documentation
- [ ]  javascript
- [ ]  functionality
- [x]  refactor

### What does this PR do? Describe your changes:

- Adds functionality:
- Updates JS for: Updated API to specify expected results for better typescript handling and made minor modifications to CSS for stylistic improvements. 
- Fixes bug for:
- Writes test for:

### Relevant Tickets?

- Finishes iteration or ticket for (type # to bring up a list of issues within the project).

### Questions/Comments/Relevant Screenshots?

Screenshot 2023-05-27 at 4 12 44 PM

### CODE REVIEW

**Review by JWFeatherstone — APPROVED** [review]
Nice, thanks Ashley! I think the one potential thing I see here is that we can probably use the same type checking for both fetchCocktails and fetchRandom - I think they're both returning an array of Drink objects, so no need to define a separate interface on lines 3-5, we can just use Promise<Drink[]>.

**Comment by hfaerber** [comment]
> Nice, thanks Ashley! I think the one potential thing I see here is that we can probably use the same type checking for both fetchCocktails and fetchRandom - I think they're both returning an array of Drink objects, so no need to define a separate interface on lines 3-5, we can just use Promise<Drink[]>.

Love this comment.  Nice work thoroughly reviewing each other's code y'all!

File: src/Components/API/apiCalls.ts [hunk 1]
```diff
@@ -1,12 +1,27 @@
...
+    } else {
+      return res.json() as Promise<RandomResponse>
+    };
+  });
 };
 
-export { fetchRandom };
\ No newline at end of file
+const fetchCocktails = async (alcohol: string): Promise<Drink[]> => {
+  const response = await fetch(`https:www.thecocktaildb.com/api/json/v1/1/filter.php?i=${alcohol}`);
```
Comment (hfaerber) [inline src/Components/API/apiCalls.ts:19]: Love seeing you dabble in async await

File: src/Components/RandomDrink/RandomDrink.tsx [hunk 2]
```diff
@@ -9,14 +9,15 @@ interface RandomDrinkProps {
...
   return (
+    <>
+    <h2>Lucky Libations</h2>
     <div className="diamond-container">
       <div className="diamond-container-inner">
-        <h2>Lucky Libations</h2>
-        <h2 className="title">{title}</h2>
-        <img src={randomImg} alt={title} id={drinkId}/>
-        <h4>Try Something New</h4>
+        <img src={randomImg} alt={title} id={drinkId} />
```
Comment (hfaerber) [inline src/Components/RandomDrink/RandomDrink.tsx:16]: Nice work including the alt tag here with helpful descriptive text like the title

**Review by hfaerber — COMMENTED** [review]
This looks like some great work y'all!  Keep it up!

**Pull request merged**